database/migrations/new/2023_03_01_013336_create_new_documents_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateNewDocumentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('new_documents', function (Blueprint $table) {
```

Comment:
We need document name and document type filled

e.g 

document_type = guarantor_id_card 

document name = "Guarnator ID Card"

you can easily set document type from the document name supplied when the document is been created

Reply:
You can have a class that holds this, where key is the document type 
```
<?php

namespace App\Utils;

class DocumentType
{
    const GUARANTOR_ID_CARD =  [
        'key' => 'guarantor_id_card',
        'name' => 'Guarantor ID Card'
    ];

    const GOODS_IN_TRANSIT_INSURANCE = [
        'key' => 'goods_in_transit_insurance',
        'name' => 'Goods In Transit Insurance'
    ];
    const FIDELITY_INSURANCE = [
        'key' => 'fidelity_insurance',
        'name' => 'Fidelity Insurance'
    ];

    const CAC_DOCUMENT = [
        'key' => 'cac_document',
        'name' => 'CAC Document'
    ];


    const TRUCK_PICTURE = [
        'key' => 'truck_picture',
        'name' => 'Truck Picture'
    ];
    const TRUCK_PROOF_OF_OWNERSHIP = [
        'key' => 'truck_proof_of_ownership',
        'name' => 'Truck Proof of Ownership'
    ];
    const TRUCK_ROAD_WORTHINESS = [
        'key' => 'truck_road_worthiness',
        'name' => 'Truck Road Worthiness'
    ];

    const TRUCK_LICENSE = [
        'key' => 'truck_license',
        'name' => 'Truck License'
    ];
    const TRUCK_INSURANCE = [
        'key' => 'truck_insurance',
        'name' => 'Truck Insurance'
    ];
}
```